deps/exokit-bindings/webglcontext/src/webgl.cc

```diff
@@ -4940,10 +4998,18 @@ NAN_METHOD(WebGLRenderingContext::GetSupportedExtensions) {
   int numExtensions = sizeof(webglExtensions)/sizeof(webglExtensions[0]);
   Local<Array> result = Nan::New<Array>(numExtensions);
   for (GLint i = 0; i < numExtensions; i++) {
-    // char *extension = (char *)glGetStringi(GL_EXTENSIONS, i);
-    result->Set(i, JS_STR(webglExtensions[i]));
+    char *extension = (char *)glGetStringi(GL_EXTENSIONS, i);
+    if(extension == webglExtensions[i]){
+      result->Set(i, JS_STR(webglExtensions[i]));
+    }
+  }
+  for (GLint i = 0; i < numExtensions; i++) {
+    string currentExtension = string(webglExtensions[i]);
+    bool hasWebGL = currentExtension.find("WEBGL");
```

Comment:
I think there are corner cases in this code. There's no guarantee that any extension called `WEBGL` will not have been already included -- in which case our list would double up.

Reply:
93391d1